glide.yaml

```diff
@@ -21,9 +21,19 @@ import:
   - package: github.com/openshift/library-go
     version: master
     # others last
-  - package: github.com/go-bindata/go-bindata
-    version: master
+  - package: github.com/jteeuwen/go-bindata
```

Comment:
Is this intentional? the repo has this message in the README: `this repository is not maintained. Questions or suggestions can be posted here.`

Reply:
Yes, tried to match what's in `library-go` https://github.com/openshift/library-go/blob/master/glide.yaml#L32-L33 and what is in `cluster-svcat-apiserver-operator` https://github.com/openshift/cluster-svcat-apiserver-operator/blob/master/glide.yaml#L21-L22